### PR TITLE
add query applications in one group

### PR DESCRIPTION
**Pull request by work-jlsun**

### CODE REVIEW

File: core/http/api/v2/application/apis.go [hunk 1]
```diff
@@ -256,6 +256,16 @@ func (a *API) List(c *gin.Context) {
 		keywords[common.ApplicationQueryByUser] = uint(id)
 	}
 
+	idStr = c.Query(common.ApplicationQueryByGroup)
+	if idStr != "" {
+		id, err := strconv.Atoi(idStr)
+		if err != nil {
+			response.AbortWithRPCError(c, rpcerror.ParamError.WithErrMsg("user id is not a number"))
```
Comment (xuzhu-591) [inline core/http/api/v2/application/apis.go:263]: user => group

File: pkg/application/dao/dao.go [hunk 2]
```diff
@@ -37,7 +37,7 @@ type DAO interface {
 	UpdateByID(ctx context.Context, id uint, application *models.Application) (*models.Application, error)
 	DeleteByID(ctx context.Context, id uint) error
 	TransferByID(ctx context.Context, id uint, groupID uint) error
-	List(ctx context.Context, groupIDs []uint, query *q.Query) (int, []*models.Application, error)
+	List(ctx context.Context, userAuthorizedGroupIDs []uint, query *q.Query) (int, []*models.Application, error)
```
Comment (xuzhu-591) [inline pkg/application/dao/dao.go:40]: Could you add ut for manager.List ?

**Comment by xuzhu-591** [comment]
lint failed

File: core/http/api/v2/application/apis.go [hunk 3]
```diff
@@ -256,6 +256,16 @@ func (a *API) List(c *gin.Context) {
 		keywords[common.ApplicationQueryByUser] = uint(id)
 	}
 
+	idStr = c.Query(common.ApplicationQueryByGroup)
```
Comment (kilosonc) [inline core/http/api/v2/application/apis.go:259]: Using array to support multiple groups?

File: pkg/application/dao/dao.go [hunk 4]
```diff
@@ -306,12 +307,14 @@ func (d *dao) List(ctx context.Context, groupIDs []uint, query *q.Query) (int, [
 				statement = statement.Where("a.template = ?", v)
 			case corecommon.ApplicationQueryByRelease:
 				statement = statement.Where("a.template_release = ?", v)
+			case corecommon.ApplicationQueryByGroup:
+				statement = statement.Where("a.group_id = ?", v)
```
Comment (kilosonc) [inline pkg/application/dao/dao.go:311]: a.group_id in ?

**Review by kilosonc — CHANGES_REQUESTED** [review]
Please take comments below into consideration

File: pkg/application/dao/dao.go (hunk 4 above)
Comment (work-jlsun) [reply pkg/application/dao/dao.go:311]: currently just  support group =

File: core/http/api/v2/application/apis.go (hunk 3 above)
Comment (work-jlsun) [reply core/http/api/v2/application/apis.go:259]: currently not support

**Review by kilosonc — APPROVED** [review]

**Review by xuzhu-591 — APPROVED** [review]

**Pull request merged**